_articles/add-a-printer-pop.md

```diff
@@ -64,7 +64,7 @@ sudo apt install --reinstall cups cups-client
 This will reinstall <u>CUPS</u>, the main printing software, which can help with generic issues.
 
 ```
-sudo apt install --reinstall ~n^system-config-printer
+sudo apt install --reinstall system-config-printer
```

Comment:
This was not an error, `~n^system-config-printer` will install any package with `system-config-printer` in the name. This includes:

```
system-config-printer
system-config-printer-gnome
system-config-printer-udev
system-config-printer-common
```

The revised command would only install `system-config-printer`. Is there a reason you would not recommend reinstalling the other three packages?

Reply:
I actually get this error when I try that command
```no matches found: ~n^system-config-printer```